test/e2e/play_kube_test.go

```diff
@@ -3535,4 +3535,19 @@ ENV OPENJ9_JAVA_OPTIONS=%q
 			Expect(kube.ErrorToString()).To(ContainSubstring("ambiguous configuration: the same config map foo is present in YAML and in --configmaps"))
 		})
 	})
+
+	It("podman play kube --log-opt = tag test", func() {
+		pod := getPod()
+		err := generateKubeYaml("pod", pod, kubeYaml)
+		Expect(err).To(BeNil())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml, "--log-driver", "journald", "--log-opt", "tag={{.ImageName}}"})
+		kube.WaitWithDefaultTimeout()
+		Expect(kube).Should(Exit(0))
+
+		start := podmanTest.Podman([]string{"start", getCtrNameInPod(pod)})
+		start.WaitWithDefaultTimeout()
+		Expect(start).Should(Exit(0))
+
```

Comment:
Please check also for the tag.  The "journald" typo suggests that the test is not sufficient.